huaweicloud/services/dataarts/resource_huaweicloud_dataarts_architecture_code_table.go

```diff
@@ -0,0 +1,758 @@
+package dataarts
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/jmespath/go-jmespath"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+// API: DataArtsStudio GET /v2/{project_id}/design/code-tables
+// API: DataArtsStudio POST /v2/{project_id}/design/code-tables
+// API: DataArtsStudio DELETE /v2/{project_id}/design/code-tables
+// API: DataArtsStudio PUT /v2/{project_id}/design/code-tables/{id}
+// API: DataArtsStudio GET /v2/{project_id}/design/code-tables/{id}
+// API: DataArtsStudio GET /v2/{project_id}/design/code-tables/{id}/values
+// API: DataArtsStudio PUT /v2/{project_id}/design/code-tables/{id}/values
+func ResourceArchitectureCodeTable() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceArchitectureCodeTableCreate,
+		ReadContext:   resourceArchitectureCodeTableRead,
+		UpdateContext: resourceArchitectureCodeTableUpdate,
+		DeleteContext: resourceArchitectureCodeTableDelete,
+
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceCodeTableImportState,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				ForceNew:    true,
+				Description: "The region where the user group is located.",
```

Comment:
Please keep consistent with the document

Reply:
fixed